AdyenSwiftUI/Present View Controller/ViewControllerPresenter.swift

```diff
@@ -72,8 +72,8 @@ import SwiftUI
                                                 context: UIViewControllerRepresentableContext<FullScreenView>) {
             if context.coordinator.currentlyPresentedViewController != nil {
 
-                dismiss(presenter: presenter, context: context) { [weak self] in
-                    self?.present(viewController: viewController, presenter: presenter, context: context)
+                dismiss(presenter: presenter, context: context) {
```

Comment:
No risk for memory leak here?

Reply:
This is a good question..
In one hand - FullScreenView(self) is a struct, so one can't pass it by reference
In the other - it will be passed somehow, (probably "boxing") that leads to a potential memory leak that we had no control over